src/db/spanner/support.rs

```diff
@@ -204,3 +211,76 @@ pub fn bso_from_row(row: Vec<Value>) -> Result<results::GetBso> {
         expiry: SyncTimestamp::from_rfc3339(&row[4].get_string_value())?.as_i64(),
     })
 }
+
+#[cfg(not(any(test, feature = "db_test")))]
+pub fn bso_to_insert_row(
+    user_id: &HawkIdentifier,
+    collection_id: i32,
+    bso: params::PostCollectionBso,
+    now: SyncTimestamp,
+) -> Result<ListValue> {
+    use super::models::DEFAULT_BSO_TTL;
+    use protobuf::well_known_types::NullValue;
```

Comment:
nit: I wonder if we should put spaces around these function `use`s just to help them stand out a bit more.

Reply:
I just switched them back up to the top, since there was the 2 left (wrapped in cfg! directives)